src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -1,17 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$(MSBuildThisFileDirectory)../../eng/Versions.props" Condition=" '$(ArcadeBuild)' != 'true' " />
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net472</TargetFrameworks>
```

Comment:
Why are we getting rid of the full framework build? In the past this was required for the task to work on desktop MSBuild.

Reply:
I added it back.

Reply:
@sbomer what is the scenario/workload for supporting .NET 4.7.2 from .NET5?

Reply:
@ericstj do you know if we will be able to unify and build MSBuild tasks for a single tfm in .NET5?

Reply:
Only if all VS/msbuild versions you support run on net5.

Reply:
This is purely about .net core build tasks, non-netcore don't even build this project

Reply:
Visual Studio always builds with .NETFramework (since VS itself is running on .NETFramework), regardless of what the project targets.  I believe this task can be used from VS, is that correct?

Reply:
Yes, it can be. Thanks for confirming.

Reply:
Wouldn't then make sense make this task netstandard2.0 to avoid dealing with multiple versions of it?

Reply:
No.  MSBuild tasks need to be runnable layouts, including dependencies specific to the target framework they are running in.  `netstandard2.0` is not runnable, dependencies will not be specific to the TFM.